app/models/provider.rb

```diff
@@ -8,6 +8,18 @@ class Provider < ApplicationRecord
   has_many :provider_users, through: :provider_users_providers
   has_many :provider_agreements
 
+  enum region_code: {
```

Comment:
For safety, is it worth matching what's in Publish? https://github.com/DFE-Digital/teacher-training-api/blob/master/app/models/concerns/region_code.rb

Reply:
Good point, I think we prefer to use strings in the DB but we can match the same keys.